CONTRIBUTING.md

```diff
@@ -36,15 +46,15 @@ W skrócie:
 * Ogranicz tytuł commita do 50 znaków
 * Linię z tytułem zaczynamy Wielką literą
 * Tytuł, tak jak w podstawówce uczono, nie posiada na końcu kropki
-* Tytuł powinien być zapisany z wykorzystaniem bezokoliczników (używamy form: REFACTOR, UPDATE, REMOVE, RELEASE, MERGE, FIX)
+* Tytuł powinien być zapisany w języku angielskim z wykorzystaniem bezokoliczników (używamy form: REFACTOR, UPDATE, REMOVE, RELEASE, MERGE, FIX)
```

Comment:
Nie było gdzieś w issues kiedyś napisane, że piszemy po polsku?

Reply:
To dotyczy commitów, rozmowa zdaje mi się że była o issue, pull requestach itp. - #95

Reply:
Ok

Reply:
Aczkolwiek czuję tu zgrzyt. Piszemy PR i Issues po polsku, a tutił commita i jego opis wymaga pisania po angielsku? Rozumiem, używać domyślnie przypisywanych tytułów z edycji live na githubie, ale zastanawiam się nad kwestią spójności. Jesteśmy polskim forum, z polskimi programistami, projekt nie jest rozwijany międzynarodowo - przynajmniej w naszym przypadku. Wszystkie zmiany wprowadzamy na potrzebę wyłącznie naszego forum i nie wspieramy oficjalnie oryginalnego projektu. Nie wiem czy jest większy sens pisać cokolwiek po angielsku. Chyba że jestem jedyny, który tak o tym myśli, to spoko